src/test/app/ValidatorSite_test.cpp

```diff
@@ -185,12 +187,24 @@ class ValidatorSite_test : public beast::unit_test::suite
         }
     };
 
+    struct fetchListConfig
+    {
+        std::string path_;
+        std::string msg_;
+        bool failFetch_ = false;
+        bool failApply_ = false;
+        int serverVersion_ = 1;
+        std::chrono::seconds expiresFromNow_ = std::chrono::seconds{3600};
```

Comment:
nit: Our format typically does not use a trailing underscore denoting private on public struct members.